tasks/5-7-filtration-unique-elem.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+function unique(arr) {
```

Comment:
This is how you should implement `unique` function with array, but you work with Set. And its possible to make it more simple. Check, what params Set takes in constructor